src/cr/cube/crunch_cube.py

```diff
@@ -449,9 +481,13 @@ def _adjust_axes(self, axes):
 
         For more info on how it needs to operate, check the unit tests.
         '''
+
+        if not self._is_axes_allowed(axes):
```

Comment:
`_is_axes_allowed` -> `_validate_axis` IMO

Reply:
Well, IDK, this is more logical to me... I know that we're not adhering to any paradigm, but in a perfect world (read _functional_) you'd have a pure function. So the name of this one tries to depict that (it will return either `True` or `False`).

`_validate_axis` could be considered something as: check if they're OK and fix them if they're not (which I tried to avoid). But if you feel strongly for `validate` approach, I don't disagree strongly (just a little bit) :)

Reply:
at least it should be `_are_axes_allowed`

I feel like we overload the word axis too much btw, I cannot say just by the name and context of this method, where this method is applicable and where not (cuz we have a lot of places that have `axis` as a param)